static/js/trailTraffic.js

```diff
@@ -129,7 +129,13 @@ var ViewModel = function() {
     map.setCenter(redwoodCenter);
   });
 
-  var segmentLines = [];
+  self.infoTooltip = ko.observable("Trailways helps horseback riders and other " +
```

Comment:
Doubt this needs to be observable, since it isn't going to change (unless that's just a KO thing).